src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -206,6 +195,65 @@
                                         Value="{x:Bind ViewModel.InitialRows, Mode=TwoWay}" />
                     </Grid>
                 </local:SettingContainer>
+
+                <!--  Launch Parameters  -->
+                <local:SettingContainer x:Uid="Globals_LaunchParameters"
+                                        CurrentValue="{x:Bind ViewModel.LaunchParametersCurrentValue, Mode=OneWay}"
+                                        Style="{StaticResource ExpanderSettingContainerStyle}">
+                    <Grid RowSpacing="10">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="Auto" />
+                        </Grid.ColumnDefinitions>
+                        <TextBlock x:Uid="Globals_LaunchMode"
+                                   Grid.Row="0"
+                                   Grid.Column="0"
+                                   VerticalAlignment="Center" />
+                        <ComboBox Grid.Row="0"

```

Comment:
Since we're not using a SettingContainer for these smaller settings anymore, make sure you test that the automation properties (name and help text) are set properly on these controls. I suspect you'll need to update Launch.cpp to get this to work properly.